README.md

```diff
@@ -920,6 +944,9 @@ arXiv, 2023.
 - <a name="Liu23tpami"></a>Liu, S., Dragotti, P.L.,    
 *[Sensing Diversity and Sparsity Models for Event Generation and Video Reconstruction from Events](https://doi.org/10.1109/TPAMI.2023.3278940)*,    
 IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2023. [Code](https://github.com/pld-group/V2E2V)
+- <a name="Qu24aaai"></a>Qu, Q., Shen, Y., Chen, X., Chung, Y.Y., Liu, T.,    
```

Comment:
The entry looks good. Thanks.

Reply:
Apologies for the inconvenience. The issue was caused by the "auto-update Table of Contents" feature in the "Markdown All in One" extension within VSCode. I have now disabled this function. Please review the pull request again. Thank you for bringing this to my attention.

Reply:
:thinking:  maybe it is not a bad feature, but we would have to fine-tune it.